backend/src/main/java/de/webdev/backend/security/AppUserDetailService.java

```diff
@@ -0,0 +1,29 @@
+package de.webdev.backend.security;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class AppUserDetailService implements UserDetailsService {
```

Comment:
https://github.com/Fabi911/Capstone-VereinsVerwaltungsapp/pull/25
```suggestion
public class AppUserDetailsService implements UserDetailsService {
```